pyarkconf/templates/schedule/schedule_edit.html

```diff
@@ -20,7 +20,7 @@ <h1>Schedule Edit</h1>
         <h2>{{ timetable.day.date }}</h2>
         {% include "schedule/_edit_grid.html" %}
     {% endfor %}
-    <form id="schedule-builder" action="." method="post" enctype="multipart/form-data">{% csrf_token %}
+    <form id="schedule-builder" action="" method="post" enctype="multipart/form-data">{% csrf_token %}
```

Comment:
Maybe `action="{{ request.path }}"`? I don't like trusting the browser to do the right thing.

Reply:
I did not review these. They were straight copy/paste from what was already here. If we decide to change it here, we'll need to change it everywhere.